db/src/androidTest/java/io/lantern/db/DBTest.kt

```diff
@@ -313,12 +310,11 @@ class DBTest {
                     assertEquals("the value", values[0].value.value)
                 }
 
-                override fun onUpdate(path: String, value: String) {
-                    assertEquals("path", path)
-                    assertEquals("new value", value)
-                }
-
-                override fun onDelete(path: String) {
+                override fun onChanges(changes: ChangeSet<String>) {
```

Comment:
ChangeSet allows us to group multiple updates and deletes together, which is handy for the UI because we can apply all of them at once and only re-render once.